README.md

```diff
@@ -270,6 +270,16 @@ The command outputs a 32-byte base64-encoded random value coming from the local
 randomness engine of the contacted server. If the encryption is not correct, the
 command outputs an error instead.
 
+### Control
+
+Unlike the randomness generation or its output, some actions or data must have restricted access. Thus the control functionalities define a set of commands that communicate on localhost to allow privacy and security. They allow the owner of a drand node to modify the running drand instance and to access their private information.
```

Comment:
`... define a set of private commands ran by the operator, and only accessible from localhost`